### PR TITLE
add apk update command to dockerfile

### PR DESCRIPTION
**Pull request by fengzhao**
please merge these requests

### CODE REVIEW

File: 0.X/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,8 @@ ENV HASHICORP_RELEASES=https://releases.hashicorp.com
 RUN addgroup consul && \
     adduser -S -G consul consul
 
+RUN apk update && apk add ca-certificates  && update-ca-certificates  && apk --no-cache add openssl wget
```
Comment (jhmartin) [inline 0.X/Dockerfile:16]: Compound RUN commands like this are usually split across lines, see the addgroup above.

**Comment by fengzhao** [comment]
please merge these requests

**Comment by woa7** [comment]
checking in the binary is wrong 0.X/consul_1.9.4_linux_amd64.zip

**Comment by fengzhao** [comment]
> checking in the binary is wrong 0.X/consul_1.9.4_linux_amd64.zip

it means that I should remove the binary  file , and then make a pull request ?

**Comment by fengzhao** [comment]
@rboyer

**Comment by fengzhao** [comment]
@ishustava

**Comment by dhiaayachi** [comment]
Hi @fengzhao,
Can you please give me more insights on the aim of this contribution? I'm looking specifically to understand:
- Is there any other reason for including `ca-certificates` in the base image? It is not used by consul to my best knowledge and could be easily added to a custom image built on top of consul if needed
- What is the binary added as part of this PR?

**Comment by kisunji** [comment]
I'm in favour of closing this PR, as the requested lines are not needed for our image and the motivations are unclear.

**Pull request closed without merging**